moveit_core/planning_scene/include/moveit/planning_scene/planning_scene.h

```diff
@@ -982,6 +982,7 @@ class PlanningScene : private boost::noncopyable, public std::enable_shared_from
   bool processCollisionObjectAdd(const moveit_msgs::CollisionObject& object);
   bool processCollisionObjectRemove(const moveit_msgs::CollisionObject& object);
   bool processCollisionObjectMove(const moveit_msgs::CollisionObject& object);
+  void normalizeObjectOrientation(const geometry_msgs::Pose& msg, Eigen::Isometry3d& out);
```

Comment:
I suggest a different name:
```suggestion
  /// convert Pose msg to Eigen::Isometry, normalizing the quaternion part if necessary.
  static void poseMsgToEigen(const geometry_msgs::Pose& msg, Eigen::Isometry3d& out);
```
And always add a doc string to new functions :wink:

Reply:
Thanks for the suggestions, I included them.